app/utils/database.py

```diff
@@ -400,6 +400,53 @@ class Meta:  # pylint: disable=too-few-public-methods
 
         indexes = ((("mother", "father", "birth_date"), True),)
 
+    def next_individual_number(herd, birth_date, breeding_event):
+        """
+        Returns the next litter number for a given year and herd.
+        """
```

Comment:
the comment and the function name sound kind of contrary to me :) what about this?
```suggestion
    def next_individual_number(herd, birth_date, breeding_event):
        """
        Returns the number for the next individual in a litter for a given year and herd.
        """
```